data/categories.txt

```diff
@@ -6093,67 +6093,651 @@ sw:Bwawa la kuogelea
+
+sport-golf
+en:Golf
+ar:الجولف
+be:Гольф
+cs:Golf
+da:Golf
+de:Golf
+el:Γκολφ
+es:Golf
+es-MX:Golf
+eu:Golfa
+fa:گلف
+fi:Golf
+fr:Le golf
+he:גולף
+hu:Golf
+id:Golf
+it:Golf
+ja:ゴルフ
+ko:골프
+mr:गोल्फ
+nb:Golf
+nl:Golf
+pl:Golf
+pt:Golfe
+pt-BR:Golfe
+ro:Golf
+ru:Гольф
+sk:Golf
+sv:Golf
+sw:Gofu
+th:กอล์ฟ
+tr:Golf
+uk:Гольф
+vi:Golf
+zh-Hans:高尔夫球
+zh-Hant:高爾夫球
+
+sport-gymnastics
+en:Gymnastics
+ar:رياضة بدنية
+be:Гімнастыка
+cs:Gymnastika
+da:Gymnastik
+de:Gymnastik
+el:Γυμναστική
+es:Gimnasia
+es-MX:Gimnasia
+eu:Gimnasia
+fa:ژیمناستیک
+fi:Voimistelu
+fr:Gymnastique
+he:התעמלות
+hu:Gimnasztika
+id:Olahraga senam
+it:Ginnastica
+ja:体操
+ko:체조
+mr:जिम्नॅस्टिक्स
+nb:Gymnastikk
+nl:Gymnastiek
+pl:Gimnastyka
+pt:Ginástica
+pt-BR:Ginástica
+ro:Gimnastică
+ru:Гимнастика
+sk:Gymnastika
+sv:Gymnastik
+sw:Gymnastics
+th:ยิมนาสติก
+tr:Jimnastik
+uk:Гімнастика
+vi:Thể dục
+zh-Hans:体操
+zh-Hant:體操
+
+sport-handball
+en:Handball
+ar:كرة اليد
+be:Гандбол
+cs:Házená
+da:Håndbold
+de:Handball
+el:Τόπι
+es:Balonmano
+es-MX:Balonmano
+eu:Eskubaloia
+fa:هندبال
+fi:Käsipallo
+fr:Handball
+he:כדוריד
+hu:Kézilabda
+id:Bola tangan
+it:Palla a mano
+ja:ハンドボール
+ko:핸드볼
+mr:हँडबॉल
+nb:Håndball
+nl:Handbal
+pl:Gra w piłkę ręczną
+pt:Handebol
+pt-BR:Handebol
+ro:Handbal
+ru:Гандбол
+sk:Hádzaná
+sv:Handboll
+sw:Mpira wa mikono
+th:แฮนด์บอล
+tr:Hentbol
+uk:Гандбол
+vi:bóng ném
+zh-Hans:手球
+zh-Hant:手球
+
+# Used to tag a scuba diving site
+sport-scuba_diving
+en:Scuba diving site
+ar:الغوص
+be:Месца для дайвінга
+cs:Potápění
+da:Scuba dykning
+de:Gerätetauchen
+el:Κατάδυση
+es:Submarinismo
+es-MX:Submarinismo
+eu:Urpekaritza
+fa:غواصی
+fi:Sukellus
+fr:Plongée sous-marine
+hu:Búvárkodás
+id:Selam scuba
+it:Immersioni in subacquea
+ja:スキューバダイビング
+ko:스쿠버 다이빙
+mr:स्कूबा डायव्हिंग
+nb:Dykking
+nl:Duiken
+pl:Nurkowanie
+pt:Mergulho
+pt-BR:Mergulho
+ro:Scufundări
+ru:Место для дайвинга
+sk:Potápanie
+sv:Dykning
+sw:Upigaji mbizi wa Scuba
+th:ดำน้ำลึก
+tr:Dalma
+uk:Місце для дайвінгу
+vi:Môn lặn
+zh-Hans:水肺潜水
+zh-Hant:水肺潛水
+
+sport-shooting
+en:Shooting
+ar:رماية
+be:Стральба
+cs:Střílení
+da:Skydning
+de:Schießen
+el:Κυνήγι
+es:Tiroteo
+es-MX:Tiroteo
+eu:Tiroketa
+fa:ورزش تیراندازی
+fi:Ammunta
+fr:Tournage
+he:ירי ספורטיבי
+hu:Lövés
+id:Penembakan
+it:Tiro
+ja:撮影
+ko:사격 경기
+mr:शूटिंग
+nb:Skyting
+nl:schieten
+pl:Strzelanie
+pt:Filmagem
+pt-BR:Filmagem
+ro:Filmare
+ru:Стрельба
+sk:Streľba
+sv:Skytte
+sw:Kupiga risasi
+th:ยิงปืน
+tr:Çekim
+uk:Стрільба
+vi:Chụp
+zh-Hans:射击
+zh-Hant:射擊
+
+sport-skiing
+en:Skiing
+ar:التزحلق
+be:Катанне на лыжах
+cs:Lyžování
+da:Stå på ski
+de:Skifahren
+el:Χιονοδρόμια
+es:Esquí
+es-MX:Esquí
+eu:Eskia
+fa:اسکی
+fi:Hiihto
+fr:Ski
+he:סקי
+hu:Síelés
+id:Bermain ski
+it:Sciare
+ja:スキー
+ko:스키 타기
+mr:स्कीइंग
+nb:Stå på ski
+nl:Skiën
+pl:Jazda na nartach
+pt:Esquiar
+pt-BR:Esquiar
+ro:Schi
+ru:Катание на лыжах
+sk:Lyžovanie
+sv:Skidåkning
+sw:Skii
+th:เล่นสกี
+tr:Kayak yapmak
+uk:Катання на лижах
+vi:Trượt tuyết
+zh-Hans:滑雪
+zh-Hant:滑雪
+
+sport-soccer
+en:Soccer
```

Comment:
en:Football
en-US:Soccer

@pastk @vng do we support it?

Reply:
yes

Reply:
ok, atm it looks like regional languages don't work in `categories.txt` at all